app/models/saved_claim/pension.rb

```diff
@@ -37,11 +37,11 @@ def send_confirmation_email
   # Send this Pension claim to the Lighthouse Benefit Intake API
   # https://developer.va.gov/explore/api/benefits-intake/docs
   # @see Lighthouse::PensionBenefitIntakeJob
-  def upload_to_lighthouse
+  def upload_to_lighthouse(current_user = nil)
     refs = attachment_keys.map { |key| Array(open_struct_form.send(key)) }.flatten
     files = PersistentAttachment.where(guid: refs.map(&:confirmationCode))
     files.find_each { |f| f.update(saved_claim_id: id) }
 
-    Lighthouse::PensionBenefitIntakeJob.perform_async(id)
+    Lighthouse::PensionBenefitIntakeJob.perform_async(id, current_user&.uuid)
```

Comment:
Any benefit to passing the entire user object, not just the uuid?

Reply:
Sidekiq [documentation on best practices](https://github.com/sidekiq/sidekiq/wiki/Best-Practices#1-make-your-job-parameters-small-and-simple) recommends using IDs over objects, as Ruby objects might not get converted into JSON correctly in the Sidekiq calling process. If we end up needing more user details within the call later, we can fetch the user within the job (ensuring that we have the most recent user data, as well).